plugins/Kaleidoscope-Colormap-Overlay/src/kaleidoscope/plugin/Colormap-Overlay.h

```diff
@@ -53,8 +53,23 @@ class ColormapOverlay : public kaleidoscope::Plugin {
     overlay_count_ = _overlay_count;
   }
 
-  // A wildcard value for a qukey that exists on every layer.
+  template<uint8_t _layer_count>
+  void configureOverlays(uint8_t const (&overlays)[_layer_count][kaleidoscope_internal::device.matrix_rows * kaleidoscope_internal::device.matrix_columns]) {
+    overlays_      = Overlay[];
+    overlay_count_ = 0;
+    for (int layer_ = 0; layer_ < _layer_count; layer_++) {
+      for (int key_index_ = 0; key_index_ < kaleidoscope_internal::device.matrix_rows * kaleidoscope_internal::device.matrix_columns; key_index_++) {
+        int8_t color_index_ = overlays[layer_][key_index_];
+        if (color_index_ != no_color_overlay) {
```

Comment:
Should I validate the lower and upper boundaries for `color_index_`?

Reply:
I think it's generally good practice to do so, yeah.